stack/templates/s3-pvc.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.global.s3Storage.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.global.s3Storage.pvcName | default (printf "%s-s3-pvc" (include "stack.fullname" .)) }}
+  {{- if .Values.global.s3Storage.namespace }}
+  namespace: {{ .Values.global.s3Storage.namespace }}
+  {{- end }}
```

Comment:
I think Argo will set the namespace of all resources, so probably don't need to create an option for this